cmd/xdp.c

```diff
@@ -0,0 +1,16 @@
+#include <uapi/linux/bpf.h>
+#include <linux/if_ether.h>
+
+BPF_HASH(pacinfo, u8, u64, 1);
+
+int FUNCNAME(struct xdp_md *ctx)
```

Comment:
mmm. Why FUNCNAME ? Couldn't you figure out a better name? Name it something like FILTER at least or find a cooler name